Yafc.UI/ImGui/ImGuiBuilding.cs

```diff
@@ -349,14 +351,14 @@ public partial class ImGui {
         public bool IsLastMouseDown(Rect rect) => rect == mouseDownRect;
 
         public void ClearFocus() {
-            if (mouseDownRect != default) {
-                mouseDownRect = default;
+            if (focusRect != default) {
+                focusRect = default;
                 Rebuild();
             }
         }
 
         public void SetFocus(Rect rect) {
```

Comment:
The call to `SetFocus` doesn't immediately seem to need to be replaced by a call to `Rebuild`, but I know there are Windows/Linux differences there. I'd delete `SetFocus` either way.

Reply:
I moved `Rebuild()` to `SetInputTextFocus()` just to be sure everything keeps working (these UI things are so/too easy to break, and we do not have any tests for them/their functionality...)